src/v3/test/integration/__snapshots__/authenticator-enroll-google-authenticator.test.tsx.snap

```diff
@@ -112,7 +113,7 @@ exports[`authenticator-enroll-google-authenticator renders form renders QR code
                     tabindex="0"
                     type="button"
                   >
-                    Set up a different way
```

Comment:
message changed?

Reply:
Yes, this was a custom SIW Next key that was created when we started the project, the updated value is the correct key.